.eslintignore

```diff
@@ -11,3 +11,5 @@ node_modules
 /styles.scss
 /styles
 /src/styles/polaris-tokens
+/scripts/analyze-custom-properties/dist
+custom-properties-data.json
```

Comment:
I'm not a fan of adding new stuff into the root. Can we put this (and analyze-custom-properties/dist is possible) into the build folder?